src/main/java/http_server/Router.java

```diff
@@ -1,48 +1,67 @@
 package http_server;
 
 import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Set;
 
 public class Router {
-    private HashMap<String, HashMap<String, Callback>> collection;
+    private HashMap<String, HashMap<String, Callback>> router;
```

Comment:
I think `routes` might be a better name for this since the class is called `Router`... Ultimately this is simply a hash of the routes, and i think of the router as the object that has all the functionality

Reply:
But if you do end up changing the name, be sure to change the function `updateRouter` to `updateRoutes`